src/highdicom/sr/templates.py

```diff
@@ -3066,7 +3066,7 @@ def __init__(
             Number of rows in pixel data frames
         pixel_data_columns: int
             Number of rows in pixel data frames
-        additional_descriptors: Sequence[highdicom.sr.value_types.ContentItem], optional
+        additional_descriptors: Union[Sequence[highdicom.sr.value_types.ContentItem], None], optional
```

Comment:
Should we also expose `ContentItem` at the package level? Would it instead make sense to use a `Union` and potentially even restrict the set of value types?

Reply:
It's very weird that this was not previously exposed before... Think it must have been an oversight. I have now exposed it at the level of the `sr` sub-package, in line with its subclasses, and updated the docstrings which all previously used the full path. (fc2fb3f)

Regarding limiting the value types, quite possibly. But I think this is best left outside the scope of this PR. I think there needs to be further work on the ImageLibrary in general.